docs/guides/chip_tool_guide.md

```diff
@@ -382,34 +394,44 @@ $ ./chip-tool basic
 This section contains a general list of various CHIP Tool commands and options,
 not limited to commissioning procedure and cluster interaction.
 
-### Interactive mode versus single command mode
+### Interactive mode versus single-command mode
 
-By default, chip-tool runs in single command mode where if any single command
-does not complete within a certain timeout period, chip-tool will exit with a
-timeout error.
+The CHIP Tool can run in one of the following modes:
 
-Example of error:
+-   Single-command mode (default) - In this mode, the CHIP Tool will exit with a
+    timeout error if any single command does not complete within a certain
+    timeout period.
```

Comment:
Maybe it's out of this PR scope, but it would be beneficial to mention here that single-command mode results in a fact that CHIP Tool establishes CASE session over and over again with every command sent. While using the interactive mode the CHIP Tool establishes CASE session with a device on the first command and then it doesn't do that for the following commands.

Reply:
I will fix in a separate PR.